packages/replay/test/fixtures/transaction.ts

```diff
@@ -77,6 +77,7 @@ export function Transaction(obj?: Partial<Event>): any {
           'Transfer Size': 1097,
           'Encoded Body Size': 797,
           'Decoded Body Size': 1885,
+          'resource.render_blocking_status': 'non-blocking',
```

Comment:
One more bikeshed convo :p

Right now [the `blocked_main_thread` span data](https://develop.sentry.dev/sdk/performance/span-data-conventions/#mobile) key is set in the mobile sdks, to get File I/O perf issues working.

 Could we re-use that? I'm fine with not - this works with me as well.

Reply:
https://github.com/getsentry/rfcs/blob/main/text/0039-sdks-report-file-IO-on-main-thread.md

Reply:
To be clear you want to re-use the same key? I don't know about that, I'd prefer more closely matching the vendor implementation we're scraping from otherwise there will be work in the product to constantly explain the differences, vs. letting these keys be self-evident

Reply:
Yup makes sense, let's keep this unique to browser!